### ISSUE TITLE
image attachments aren't in correct format

### ISSUE THREAD

**Issue by ericmigi** [Sorunome/mx-puppet-skype#11]
when you upload a png in riot/matrix to a skype-bridged room, the image shows up in skype but the format does not allow the other skype user to view it correctly or download it.

a normal skype png image attachment: 

![image](https://user-images.githubusercontent.com/1048265/83787432-a2f45880-a648-11ea-9a34-b135c19f5aa8.png)

png image attachment from matrix bridged: 

![image](https://user-images.githubusercontent.com/1048265/83787458-ad165700-a648-11ea-8250-969f4568b420.png)

**Comment by Sorunome** [comment]
Yeah, soru can'tfigure out what she is missing that makes them look different :/ help would be great

Am 4. Juni 2020 18:49:56 MESZ schrieb Eric Migicovsky <notifications@github.com>:

**Comment by ericmigi** [comment]
if you have the bridge set up now, it should be reproducible by uploading a png in regular skype app, then uploading the same png in a bridged room from matrix. the issue is that when you send it via the bridge, the image cannot be opened (either full screen in that window, or in a new tab)

**Comment by Sorunome** [comment]
Yes, soru is aware of the issue, but she doesn't know how to fix it (and spent trying to many hours in the past), thus asking for help from others

Am 4. Juni 2020 19:20:54 MESZ schrieb Eric Migicovsky <notifications@github.com>:

**Comment by ericmigi** [comment]
oooh I get it.
